cfg/1.13/master.yaml

```diff
@@ -185,6 +185,8 @@ groups:
   - id: 1.1.12
     text: "Ensure that the admission control plugin DenyEscalatingExec is set (Scored)"

```

Comment:
I think it’s worth also editing the text field to add “[DEPRECATED]” like the benchmark document

Reply:
Done